andela/AndelaEatsUI
# #161761135 Refactor vendor input

#### What does this PR do?
- refactor vendor input fields

#### Description of Task to be completed?
- refactor vendor input fields

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ch-refactor-vendor-input-161761135`
- run these commands `npm run bundle` and `npm run dev` 
- run the command `npm run test-dev` to ensure all tests are passing

#### What are the relevant pivotal tracker stories?
- [161761135](https://www.pivotaltracker.com/n/projects/2178749/stories/161761135)

#### Background Context
Nil

#### Screenshots (If Applicable)
Nil
